.github/workflows/ci.yml

```diff
@@ -83,3 +75,28 @@ jobs:
 
     # Fail if any changes were written to source files (ex, from: build/build-cdt-lib.js).
     - run: git diff --exit-code
+
+  smoke:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        invert: [false, true]
```

Comment:
also amusing would be
```yml
matrix:
  smoketest: [a11y, errors, oopif, pwa, pwa2, pwa3, dbw, redirects, seo, offline, byte, perf, lantern, metrics, legacy-javascript, source-maps]
# ...
- name: Run smoke test
  run yarn smoke -j=1 --retries=2 ${{ matrix.smoketest }}
```
and launch all the worker jobs :)

But I'm pretty sure we [compete with the rest of the org for workers](https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits)?